README.md

```diff
@@ -1,4 +1,4 @@
-# Echo Server
+# Echo echo.Server
```

Comment:
Missing information on how to run and test it.

Reply:
done ✅